src/offset/local/windows.rs

```diff
@@ -67,12 +68,14 @@ fn tm_to_datetime(mut tm: Tm) -> DateTime<Local> {
 
     let date = NaiveDate::from_ymd_opt(tm.tm_year + 1900, tm.tm_mon as u32 + 1, tm.tm_mday as u32)
         .unwrap();
-    let time = NaiveTime::from_hms_nano(
+
+    let time = NaiveTime::from_hms_nano_opt(
         tm.tm_hour as u32,
         tm.tm_min as u32,
         tm.tm_sec as u32,
         tm.tm_nsec as u32,
-    );
+    )
+    .expect("invalid time");
```

Comment:
this could be cast to a `LocalResult::None` which would avoid the `expect` call (this is semantically sub-optimal, but is consistent with usage elsewhere):

```rust
let time = NaiveTime::from_hms_nano_opt(...).ok_or(LocalResult::None)?;
```

Reply:
Pushed changes (will squash later).